proto/connector_service.proto

```diff
@@ -58,13 +58,14 @@ message SinkWriterStreamRequest {
       bytes binary_data = 1;
     }
 
+    uint64 batch_id = 1;
+    uint64 epoch = 2;
+
     oneof payload {
-      JsonPayload json_payload = 1;
-      StreamChunkPayload stream_chunk_payload = 2;
+      JsonPayload json_payload = 128;
+      StreamChunkPayload stream_chunk_payload = 129;
+      int64 stream_chunk_ref_pointer = 130;
```

Comment:
Any reason why the index starts from 128?

Reply:
It can be any random value that is high enough. I just realize that the index of items in `oneof` will occupy the index of outer message. Previously, the two payload occupies 1 and 2, and the later `batch_id` and `epoch` occupies 3 and 4, and now when we want to add new item to the `oneof`, we should set its index as 5, which is not next to the original payloads. This is why I change the index, though it's very wired.